src/client/object/AppendForm.js

```diff
@@ -539,16 +539,22 @@ export default class AppendForm extends Component {
     const { intl, wObject, form } = this.props;
     const currentField = form.getFieldValue('currentField');
     const currentLocale = form.getFieldValue('currentLocale');
+    const filds = form.getFieldsValue();
     const filtered = wObject.fields.filter(
       f => f.locale === currentLocale && f.name === currentField,
     );
+    const triggerValue = trimStart(form.getFieldValue(currentField)).replace(/\s{2,}/g, ' ');
```

Comment:
Is 'form.getFieldValue(currentField)' the same as 'filds[currentField]'? Better to use second one